l10n_br_account_banking_payment_cnab/febraban/cnab_240/cnab_240.py

```diff
@@ -49,9 +52,23 @@ def get_bank(bank):
         elif bank == '104':
             from bancos.cef import Cef240
             return Cef240
+        elif bank == '033':
+            from bancos.santander import Santander240
+            return Santander240
         else:
```

Comment:
This sequence of `if ... elif` clauses should be replaced by a dictionary defined at the module level, e.g.:
    BANK_MAP = {
        '341': bancos.itau.Itau240,
        '104': bancos.bradesco.Bradesco240,
        ...
    }

Even better would be to put the bank number as an attribute each `Cnab240` subclass, and make the `Cnab240` class belong to a metaclass that populates this dictionary automatically when a subclass of `Cnab240` is generated.